backend/helpers/event_cleanup_scheduler.py

```
@@ -39,19 +40,29 @@ async def event_cleanup_loop(db):
             
+            # Perform refund cleanup operations
+            refund_results = await _cleanup_stale_refunds(db)
+            
             unpublished = results.get('unpublished', 0)
             cleaned_recurring = results.get('cleaned_recurring', 0)
             total_rsvps_removed = results.get('total_rsvps_removed', 0)
             
-            if unpublished > 0 or cleaned_recurring > 0:
+            refund_completed = refund_results.get('completed', 0)
+            refund_rolled_back = refund_results.get('rolled_back', 0)
+            refund_failed = refund_results.get('failed', 0)
```

Ruff (0.14.2)

Local variable `refund_failed` is assigned to but never used

Remove assignment to unused variable `refund_failed`

(F841)